supernode/daemon/mgr/progress/state_sync_map.go

```diff
@@ -1,47 +1,45 @@
 package progress
 
 import (
-	"sync"
-
 	"github.com/dragonflyoss/Dragonfly/pkg/errortypes"
-	"github.com/dragonflyoss/Dragonfly/pkg/stringutils"
+	"github.com/dragonflyoss/Dragonfly/pkg/syncmap"
 
 	"github.com/pkg/errors"
 )
 
 // stateSyncMap is a thread-safe map.
 type stateSyncMap struct {
-	*sync.Map
+	smap *syncmap.SyncMap
```

Comment:
you could still define the field as `syncmap`, otherwise it is a little strange in main code bock to se smap. Somewhat hard to image syncmap from smap. 😄